scalyr_agent/platform_windows.py

```diff
@@ -418,10 +421,10 @@ def run_as_user(self, user_name, script_file, script_binary, script_arguments):
             )
 
         if not self.__no_escalation_warning:
-            print "The process requires Administrator permissions to complete this action."
-            print "Attempting to escalate privileges, which will require user confirmation or the Administrator "
-            print "password through a dialog box that is about to be shown."
-            raw_input("Hit Enter to continue and view the dialog box.")
+            print("The process requires Administrator permissions to complete this action.")
+            print("Attempting to escalate privileges, which will require user confirmation or the Administrator ")
+            print("password through a dialog box that is about to be shown.")
+            input("Hit Enter to continue and view the dialog box.")
```

Comment:
Do you know the difference between `raw_input` and `input`?

Reply:
In python3 input acts like raw_input. To keep this in python2, modernize imports six.moves.input which is an alias of raw_input.